kcidb/misc.py

```diff
@@ -36,7 +36,7 @@
 }
 
 # Check light assertions only, if True
-LIGHT_ASSERTS = not os.environ.get("KCIDB_HEAVY_ASSERTS", "")
+LIGHT_ASSERTS = not os.environ.get("KCIDB_HEAVY_ASSERTS", "Not empty")
```

Comment:
Could you please revert this change? This would make heavy assertions always enabled and our production deployment **very** slow. Please be careful to avoid submitting such changes. I could've easily overlooked it.

If you want to run tests, just execute `export KCIDB_HEAVY_ASSERTS=1` in your shell before that.

Reply:
Oh, This was a mistake 
I added it because the test was failing  😃 
WIll revert back, thanks.